rcl_interfaces/msg/ParameterValue.msg

```diff
@@ -6,8 +6,8 @@
 uint8 type
```

Comment:
My reasoning for not putting a default for this was that, while all-but-one of the `_value` fields will be unused, the `type` is not "optional" so the client library should always initialise it itself